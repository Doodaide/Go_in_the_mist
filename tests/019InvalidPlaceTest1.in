who
hist
who 
WHO
wHo
Hello World 
history view
Bruh Moment 
whowhowhowhowhowhowhowho
history history history history 
place place place place
whowhowhowhowh\nwhowhowho
whowhowhowhowho\nwhowhowh
whowhowhowhowh\0whowhowho
whowhowhowhowho\0whowhowho

who
term