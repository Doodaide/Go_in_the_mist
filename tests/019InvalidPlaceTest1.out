B
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
B
